cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/TypeManager.java

```diff
@@ -317,7 +296,7 @@ public void setLanguageFrontend(@NotNull LanguageFrontend frontend) {
   }
 
   public boolean isPrimitive(Type type) {
-    return primitiveTypeNames.contains(type.getTypeName());
+    return getLanguage().getPrimitiveTypes().contains(type.getTypeName());
```

Comment:
Can we add language as an additional parameter here? I want to get rid of the `getLanguage` calls in the singleton type manager.